manifests/t/the-sz/Trion/1.10/the-sz.Trion.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.Trion
+PackageVersion: 1.10

```

Comment:
The package version does not appear in the ARP entry. This will cause an upgrade loop
![image](https://user-images.githubusercontent.com/12611259/125691005-04781d5a-3cf5-4970-a308-e632d37aa4c2.png)

Reply:
fixed with new commit